frontend/manage_herds.tsx

```diff
@@ -0,0 +1,96 @@
+/**
+ * @file This file contains the Manage function. This function is used for
+ * granting and revoking permissions from users, as well as approving requests
+ * for adding individuals to herds in the genebanks you manage.
+ */
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import { Tabs, Tab } from '@material-ui/core/'
+import { Box, Button } from '@material-ui/core'
+import { HerdForm } from '~herdForm'
+import { Genebank } from 'data_context_global'
+
+import { useDataContext } from './data_context'
+
+// Define styles for tab menu
+const useStyles = makeStyles({
+  sidebar: {
+    float: "left",
+    height: "calc(100% - 50px)",
+    width: "250px",
+    margin: "0",
+    borderRight: `1px solid rgba(0,0,0,0.2)`,
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center"
+  },
+  verticalTabs: {
+    height: "calc(100% - 50px)",
+    width: "250px",
+  },
+  controls: {
+    height: "100%",
+    width: "calc(100% - 250px)",
+    padding: "0.5cm 1cm",
+    overflowY: "scroll",
+  },
+  centerButton: {
+    width: "80%",
+  }
+});
+
+export function ManageHerds({id}: {id: number | undefined}) {
```

Comment:
There is a lot duplicated here and in `ExploreHerds`. Why not just add an edit button when Exploring if you're an admin?

Reply:
yeah - it wasn't originally supposed to be so similar, but I ran out of time. so I'll have to think about the final design before I decide if there's value in optimizing.

Reply:
One option is just to write this down in an issue and let it be like it is right now of course.